back-end/helpers/db-helpers.js

```diff
@@ -0,0 +1,12 @@
+const mysql = require('mysql');
+let pool = mysql.createPool({ connectionLimit : 10, host : 'localhost', database : 'test', user : 'root', password : 'root', port : '8889' });
```

Comment:
One last thing can you add a comment stating that the configuration is dependent on user's mysql/db setup?

Reply:
Yea no prob :+1: